pkg/cloud/services/network/subnets.go

```diff
@@ -428,11 +428,11 @@ func (s *Service) getSubnetTagParams(unmanagedVPC bool, id string, public bool,
 	// Add tag needed for Service type=LoadBalancer
 	additionalTags[infrav1.NameKubernetesAWSCloudProviderPrefix+s.scope.KubernetesClusterName()] = string(infrav1.ResourceLifecycleShared)
 
-	for k, v := range manualTags {
```

Comment:
`additionalTags := s.scope.AdditionalTags()` in line 418 already adds the additional tags from `s.AWSCluster.Spec.AdditionalTags`. 

That is what we are trying to avoid.

I don't know if manual tags are also relevant here, guessing not.

Reply:
Done

Reply:
For safer side, I have removed manual tags as well from unmanaged subnets.

Reply:
And what was in the manual tags?

Reply:
Manual tags are existing tags on that subnet which is fetched from scope, I think we need to retain it here, wdyt @sedefsavas ?

Reply:
Any views here @sedefsavas ?

Reply:
Our networking code really needs a complete re-write!

Reply:
I agree 😞

Reply:
I can't see this change causing issues.